src/data/JS/highlight-hover.js

```diff
@@ -0,0 +1,28 @@
+var highlight = function (on) {
```

Comment:
Add copyright notice from Haddock project. Check that the file indeed comes from there.

Reply:
I modified it a little, not sure how does that affect the copyright though.

Reply:
Added the copyright line and a link to the license